assets/src/utils/common_utils.py

```diff
@@ -335,7 +394,7 @@ def is_blank_is_created(self) -> bool:
             get_pages(file_to_upload),
             schemas.FileProcessingStatus.UPLOADING,
         )
-        if ext == ".txt":
+        if ext in (".txt", ".html"):
```

Comment:
Maybe you will add all gotenberg formats here?

Reply:
I think we should open new task for this feature.